bobtimus/src/amounts.rs

```diff
@@ -5,7 +5,7 @@ use rust_decimal::{
     Decimal, RoundingStrategy,
 };
 use serde::{Deserialize, Serialize};
-use std::{convert::TryFrom, fmt::Debug};
+use std::{convert::TryFrom, fmt, fmt::Debug};
```

Comment:
I tend to always import just the `fmt` module and refer to everything else from there. Makes it more consistent and reduces the number of imports.

The thing is, you will always need `fmt::Fromatter` and `fmt::Result` as well.